templates/chat/roll-dialog.hbs

```diff
@@ -4,6 +4,14 @@
         <label>{{ localize "DND5E.Formula" }} {{#if type}}({{type}}){{/if}}</label>
         <input type="text" name="formula" value="{{formula}}" disabled>
     </div>
+    {{#if types}}
+    <div class="form-group">
+        <label>{{ localize "DND5E.DamageType" }}</label>
+        <select name="roll.{{ @index }}.type">
```

Comment:
I imagine that in most cases, there's only 1 damage type, so I don't know if we really need to show a drop-down all the time. I suppose the alternative would be to just not show the damage type at all if there's only one, or show a readonly field or something. Both of those options are more work though so maybe it's not worth it. I imagine this dialog might get some updates, so maybe we revisit it then.